source/recipe-individual/scripts/individual.js

```diff
@@ -0,0 +1,28 @@
+window.addEventListener('DOMContentLoaded', init);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
window.addEventListener("DOMContentLoaded", init);
```